be/src/exec/jni_scanner.h

```diff
@@ -31,15 +31,14 @@ class JniScanner : public HdfsScanner {
     ~JniScanner() override { close(); }
 
     [[nodiscard]] Status do_open(RuntimeState* runtime_state) override;
-    void do_update_counter(HdfsScanProfile* profile) override;
+    void do_update_counter(HdfsScanProfile* profile) override {}
     void do_close(RuntimeState* runtime_state) noexcept override;
     Status do_get_next(RuntimeState* runtime_state, ChunkPtr* chunk) override;
     [[nodiscard]] Status do_init(RuntimeState* runtime_state, const HdfsScannerParams& scanner_params) override;
     bool is_jni_scanner() override { return true; }
 
 protected:
-    [[nodiscard]] Status fill_empty_chunk(RuntimeState* runtime_state, ChunkPtr* chunk,
-                                          const std::vector<SlotDescriptor*>& slot_desc_list);
+    [[nodiscard]] Status fill_empty_chunk(ChunkPtr* chunk, const std::vector<SlotDescriptor*>& slot_desc_list);
```

Comment:
all nodiscard can be removed because we have add nodiscard keyword before Status class